acbuild/acbuild.go

```diff
@@ -153,7 +153,7 @@ func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd
 
 		cmdExitCode = cf(cmd, args)
 
-		err = a.Write(aciToModify, true, false, nil)
+		err = a.Write(aciToModify, true, false, false, nil)
```

Comment:
I had thought about this but wasn't sure... I would agree with compressing them by default. It seems more ideal.